civihr_default_theme/scss/components/_ui-select.scss

```diff
@@ -76,6 +76,17 @@
 * TEMP fixes till ui-select styling is fixed
 * Dropdown ul opacity was set to 0
 */
-.chr_leave-request-modal__form .ui-select-container ul {
-  opacity: 1 !important;
+.chr_leave-request-modal__form {
+ .ui-select-container ul {
```

Comment:
as you can see in all other selectors the first selector is below parent and rest of them have line breaks like 

```
.ui-select-bootstrap .ui-select-choices-row {
  > span {
```
```
.ui-select-container {
  &:not(.ui-select-multiple) {
```
this is just consistent with them